.vs/config/applicationhost.config

```diff
@@ -0,0 +1,1023 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Oh...it's been checked in yesterday. and I didn't include this file. :-) Thanks!